README.md

```diff
@@ -97,12 +97,3 @@ It's a miracle. Get it at http://www.graylog2.org/
 ## Installation
 
     npm install graylog2
-
-## License
```

Comment:
Could we link to the LICENSE file instead then? Something like 

See the (license file)[./LICENSE]

?